modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -106,6 +106,27 @@ Your username must be at least three characters and contain only the following:
 [NOTE]
 You cannot create a new account within an existing organization without an email invitation.
 
+== Sign in to Anypoint Platform
+
+. Navigate to the Anypoint Platform signin screen.
+. Enter your username and password.
+. Click *Sign in*.
+
+[NOTE]
+If you try to sign in and Anypoint Platform says your username and password are incorrect when you have entered the correct credentials, your organization may use an xref:access-management::external-identity.adoc[external identity provider] to initiate the sign in process. You can either use your external identity provider to sign you in, or you can xref:access-management::managing-your-account.adoc#use-SPISSO[enter your organization's custom domain name] prior to entering your credentials. 
+
+[[use-SPISSO]]
+== Sign in to Anypoint Platform using Service Provider-Initiated Single Sign-on
+
+If your organization uses xref:access-management::external-identity.adoc[service-provider initiated single sign-on], you must provide your organization's custom domain name as part of the signin process.
+
+. Navigate to the Anypoint Platform signin screen.
+. Click *Use custom domain*.
+. Enter your organization's custom domain name.
+. Click *Continue*.
+. Enter your username and password.
```

Comment:
This is not the case, they will have an option to continue to their IdP where they will do their idp login and will get redirected to anypoint platform or login with anypoint if the user is a non-federated user as the image below, the image is just for your reference don't add here.
![image](https://user-images.githubusercontent.com/2031280/80835195-1ff26500-8ba7-11ea-8192-156bb25d6142.png)

Reply:
Right, the just click "Continue".  No need to specify a username or password (that will be handled by their IDP)